terraform/vars/tvm-ci-prod.auto.tfvars

```diff
@@ -38,6 +38,15 @@ autoscaler_types = {
     on_demand_percentage_above_base_capacity = 100
     on_demand_base_capacity                  = 0
   }
+  "Prod-Autoscaler-Jenkins-CPU-SPOT" = {
```

Comment:
we don't really use this instance type for anything, it was originally the main CPU instance but the `r5.large`s are 10x cheaper and only 1-2x slower. I'm in favor of just deleting this and the `CPU` autoscaling group instead since I see there are some usages in TVM today (which we would undo first)